src/Legacy/Storage.php

```diff
@@ -785,6 +796,13 @@ private function searchSingleContentType($query, $contenttype, $fields, array $f
         return $results;
     }
 
+    protected function buildSearchQuery($table, $taxonomytable, $where)
+    {
+        $where = implode(' AND ', $where);
+
+        return "SELECT $table.id FROM $table LEFT JOIN $taxonomytable ON $table.id = $taxonomytable.content_id LEFT JOIN bolt_field_value ON $table.id = bolt_field_value.content_id WHERE $where GROUP BY $table.id";
```

Comment:
Um, 215 characters … This is much less readable, breaks coding standard (PSR-2), expands the API unnecessarily.

Reply:
Ok - fixed.